pom.xml

```diff
@@ -519,11 +520,6 @@
                 <version>${kubernetes-client.version}</version>
             </dependency>
 
-            <!--
-              because of THRIFT-4805, we don't upgrade to libthrift:0.12.0,
-              because of THRIFT-5274, we don't upgrade to libthrift:0.13.0,
-              so just keep hive-service-rpc:2.3.9 transitive dependency libthrift:0.9.3
-            -->
```

Comment:
Then we can exclude *:* from `hive-service-rpc`, and add fb303 and thrift explicitly